doc/nrf/config_and_build/configuring_app/sysbuild_images.rst

```diff
@@ -242,4 +242,4 @@ Kconfig.sysbuild:
 Next steps
 ==========
 
-For more information on sysbuild, see :ref:`zephyr:sysbuild`, :ref:`zephyr_samples_sysbuild`, and :ref:`sysbuild_forced_options`.
+For more information on sysbuild, see the :ref:`sysbuild_forced_options` page in the |NCS| documentation and :ref:`zephyr:sysbuild` and :ref:`zephyr_samples_sysbuild` documentation pages in Zephyr.
```

Comment:
IIRC `zephyr_samples_sysbuild` is from NCS, not Zephyr

Reply:
Lemme fix